semigraphics.go

```diff
@@ -0,0 +1,234 @@
+	BoxDrawingsUpDoubleAndRightSingle             rune = '\u2559' // ╙
+	BoxDrawingsDoubleUpAndRight                   rune = '\u255a' // ╚
+	BoxDrawingsUpSingleAndLeftDouble              rune = '\u255b' // ╛
+	BoxDrawingsUpDobuleAndLeftSingle              rune = '\u255c' // ╜
+	BoxDrawingsDoubleUpAndLeft                    rune = '\u255d' // ╝
+	BoxDrawingsVerticalSingleAndRightDouble       rune = '\u255e' // ╞
+	BoxDrawingsVerticalDoubleAndRightSingle       rune = '\u255f' // ╟
+	BoxDrawingsDoubleVerticalAndRight             rune = '\u2560' // ╠
+	BoxDrawingsVerticalSingleAndLeftDouble        rune = '\u2561' // ╡
+	BoxDrawingsVerticalDoubleAndLeftSingle        rune = '\u2562' // ╢
+	BoxDrawingsDoubleVerticalAndLeft              rune = '\u2563' // ╣
+	BoxDrawingsDownSingleAndHorizontalDouble      rune = '\u2564' // ╤
+	BoxDrawingsDownDoubleAndHorizontalSingle      rune = '\u2565' // ╥
+	BoxDrawingsDoubleDownAndHorizontal            rune = '\u2566' // ╦
+	BoxDrawingsUpSingleAndHorizontalDouble        rune = '\u2567' // ╧
+	BoxDrawingsUpDoubleAndHorizontalSingle        rune = '\u2568' // ╨
+	BoxDrawingsDoubleUpAndHorizontal              rune = '\u2569' // ╩
+	BoxDrawingsVerticalSingleAndHorizontalDouble  rune = '\u256a' // ╪
+	BoxDrawingsVerticalDoubleAndHorizontalSingle  rune = '\u256b' // ╫
+	BoxDrawingsDoubleVerticalAndHorizontal        rune = '\u256c' // ╬
+	BoxDrawingsLightArcDownAndRight               rune = '\u256d' // ╭
+	BoxDrawingsLightArcDownAndLeft                rune = '\u256e' // ╮
+	BoxDrawingsLightArcUpAndLeft                  rune = '\u256f' // ╯
+	BoxDrawingsLightArcUpAndRight                 rune = '\u2570' // ╰
+	BoxDrawingsLightDiagonalUpperRightToLowerLeft rune = '\u2571' // ╱
+	BoxDrawingsLightDiagonalUpperLeftToLowerRight rune = '\u2572' // ╲
+	BoxDrawingsLightDiagonalCross                 rune = '\u2573' // ╳
+	BoxDrawingsLightLeft                          rune = '\u2574' // ╴
+	BoxDrawingsLightUp                            rune = '\u2575' // ╵
+	BoxDrawingsLightRight                         rune = '\u2576' // ╶
+	BoxDrawingsLightDown                          rune = '\u2577' // ╷
+	BoxDrawingsHeavyLeft                          rune = '\u2578' // ╸
+	BoxDrawingsHeavyUp                            rune = '\u2579' // ╹
+	BoxDrawingsHeavyRight                         rune = '\u257a' // ╺
+	BoxDrawingsHeavyDown                          rune = '\u257b' // ╻
+	BoxDrawingsLightLeftAndHeavyRight             rune = '\u257c' // ╼
+	BoxDrawingsLightUpAndHeavyDown                rune = '\u257d' // ╽
+	BoxDrawingsHeavyLeftAndLightRight             rune = '\u257e' // ╾
+	BoxDrawingsHeavyUpAndLightDown                rune = '\u257f' // ╿
+)
+
+// SemigraphicJoints is a map for joining semigraphic (or otherwise) runes.
+// So far only light lines are supported but if you want to change the border
+// styling you need to provide the joints, too.
+var SemigraphicJoints = map[string]rune{
```

Comment:
Have you tested all of these joints? Or did you generate them from the previous joint list? I want to be sure that these combinations are correct.

Reply:
I "search&replace"-d them, and the table demo looked good. But I will check again and maybe add comments with the cimbined runes for a better understanding.